src/Jackett.Common/Indexers/CardigannIndexer.cs

```diff
@@ -764,9 +764,7 @@ protected string getRedirectDomainHint(string requestUrl, string RedirectUrl)
             return null;
         }
 
-        protected string getRedirectDomainHint(WebClientByteResult result) => getRedirectDomainHint(result.Request.Url, result.RedirectingTo);
-
-        protected string getRedirectDomainHint(WebClientStringResult result) => getRedirectDomainHint(result.Request.Url, result.RedirectingTo);
+        protected string getRedirectDomainHint(WebResult result) => getRedirectDomainHint(result.Request.Url, result.RedirectingTo);
```

Comment:
In retrospect, changes like this should have been made even before merging the two classes. These properties are in the base class, and there was no reason to duplicate the function for the two derived classes.